### DISCUSSION TITLE
Add to Documentation a example on how to make a simple BLE-enabled app that uses BLE. Emits signal from the watch.

### DISCUSSION THREAD

**Discussion by VeryUsual** [InfiniTimeOrg/InfiniTime#1852] [General]
Add to Documentation a example on how to make a simple BLE-enabled app that uses BLE. Emits signal *from* the watch.

**Comment by JF002** [comment]
We unfortunately do not have such docs right now. Alternatively, you can have a look at the code of the [Music app](https://github.com/InfiniTimeOrg/InfiniTime/blob/main/src/displayapp/screens/Music.cpp#L224) which does send data to the companion app to change the volume, play/pause,...
